BREAKING-CHANGES.md

```diff
@@ -15,6 +15,7 @@ This is a quick summary of the sections below:
   - Removed deprecated `Block::title_on_bottom`
   - `Line` now has an extra `style` field which applies the style to the entire line
   - `Block` style methods cannot be created in a const context
+  - `Table::new` now accepts `IntoIterator<Item: Into<Row<'a>>>`.
```

Comment:
Also need to add a full description of this below (around line 49)

Reply:
Can do, and updated!